docs/fundamentals/code-analysis/quality-rules/ca1062.md

```diff
@@ -50,7 +50,7 @@ You can suppress a warning from this rule if you are sure that the dereferenced
 
 ## Configurability
 
-This rule has the following configurable options.
+This rule has the following configurable options that can be configured for just this rule, for all rules, or for all rules in this category (Design). For more information, see [Code quality rule configuration options](../code-quality-rule-options.md).
```

Comment:
It won't be used by anything else but technically you could declare it and it would work for this rule. If later on, another rule, was to reuse this option then it would pick-it up automatically.

So somehow it makes sense. Maybe instead of being repeated in all rules there should be some global section about this.

Reply:
Ping @mavasani